R/correlation.R

```diff
@@ -377,11 +393,19 @@ Correlation <- function(jaspResults, dataset, options){
       errors <- FALSE
       
       if(method != "pearson"){
-        result$conf.int <- .createNonparametricConfidenceIntervals(x = x, y = y, obsCor = result$estimate,
-                                                                   hypothesis = alternative, confLevel = conf.level,
-                                                                   method = method)
+        normal.approx.ci <- switch(method,
+                                   "spearman" = sample.size > 1290,
```

Comment:
Is the threshold 1290 based on something particular? Could you please add a note on this in the help file?

Reply:
My bad, I wasn't reading well enough, as you already mentioned that you do what cor.test does, so that's nice.